app/src/main/res/values/strings.xml

```diff
@@ -10,8 +10,17 @@
 	<string name="title_activity_search">Search</string>
 	<string name="title_activity_tours">Tours</string>
 	<string name="title_activity_favorites">Favorites</string>
+	<string name="title_activity_auth">Authorization</string>
+	<string name="title_sign_up_activity">Sign Up</string>
 
+	<!--Constants-->
+	<string name="google_server_client_id">1068705484895.apps.googleusercontent.com</string>
```

Comment:
What are these ids?

Reply:
OAuth 2.0 client ID (https://console.developers.google.com/apis/credentials/oauthclient/1068705484895.apps.googleusercontent.com?project=twobits.cz:api-project-1068705484895)

Reply:
I do not care from where iw was taken, but definitely here should not be our official app!

Reply:
So your point is that I should have created new Client ID for SDK app ?

Reply:
Moving to `strings-non-vcs.xml`